### DISCUSSION TITLE
速すぎる操作するとストックがされない

### DISCUSSION THREAD

**Discussion by misogihagi** [increments/qiita-discussions#907] [要望・アイデア・不具合]
もし既出でしたらごめんなさい

以下の手順でストックしようとすると目的のカテゴリーで保存されないです。

1. 記事をストックしようとしてストックボタンが押す
2. 自動的に一番上のカテゴリーが選択される
3. 一番上のカテゴリーで保存する意図はないので一番上のカテゴリーのチェックを外す
4. 目的のカテゴリーでチェックする
5. 更新するなどして同じ記事を見てみるとストックされていない

個人的には一番上のカテゴリーがDB上でロックされている間に他のカテゴリーも入れようとして失敗したのかなと思っています。
スマホでやると少しやりやすいです。
わからないところがあればよろしくおねがいします。

**Comment by ohakutsu** [comment]
@misogihagi 
ご報告ありがとうございます！

弊社で確認したところ再現できませんでした :bow:
また、ご指摘いただいたようなエラーが発生していないかも含めて調査してみましたが、エラーを確認できませんでした。

もし継続して同じ問題が起きているようでしたら、調査を進めるにあたり下記について教えていただけると幸いです 🙏 

- OS
- ブラウザの種類とバージョン
